karma.conf.js

```diff
@@ -0,0 +1,114 @@
+process.env.SAUCE_USERNAME = 'shard'
+var im_lazy_dot_tiff = '372040cd-d46b-4a2d-9d8b-cab91ef9efaa'
+process.env.SAUCE_ACCESS_KEY =  im_lazy_dot_tiff //shh, don't tell anyone
+
+module.exports = function(config){
```

Comment:
Can do, also since github preview is showing off my 'secure code', could we get the two variables (SAUCE_USERNAME, SAUCE_ACCESS_KEY) added to CircleCI?